pom.xml

```diff
@@ -20,14 +20,14 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.dremio.plugin</groupId>
-  <version>4.2.2-202004211133290458-b550b6fa</version>
+  <version>4.7.3-202008270723550726-918276ee</version>
   <artifactId>dremio-hbase-plugin</artifactId>
 
   <name>Dremio HBase Community Connector</name>
 
   <properties>
     <hbase.TestSuite>**/HBaseTestsSuite.class</hbase.TestSuite>
-    <version.dremio>4.2.2-202004211133290458-b550b6fa</version.dremio>
+    <version.dremio>4.7.3-202008270723550726-918276ee</version.dremio>
```

Comment:
let's move this to ${project.version} as opposed to duplicating.

Reply:
Done.